src/Microsoft.DotNet.ApiSymbolExtensions/AssemblySymbolLoader.cs

```diff
@@ -254,6 +254,12 @@ public IEnumerable<IAssemblySymbol> LoadMatchingAssemblies(IEnumerable<IAssembly
             return matchingAssemblies;
         }
 
+        /// <inheritdoc />
+        public IEnumerable<MetadataReference> GetMetadataReferences()
```

Comment:
this can just be an expression based body.
```C#
public IEnumerable<MetadataReference> GetMetadataReferences() =>  _cSharpCompilation.References;
```

Reply:
Nice. Didn't know that interfaces could have a property member. I have this https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/interface-properties as a reference.